packages/core/src/actions.ts

```diff
@@ -69,6 +69,17 @@ export function resolveActionObject(
         return [state, a];
       }
     );
+  } else if (isArray(dereferencedAction)) {
+    return createDynamicAction(
+      { type: ActionTypes.ActionGroup, params: actionObject.params ?? {} },
```

Comment:
Should we even bother with `actionObject.params` here? I had trouble wrapping my mind around this part. Would there be a reason we might want them available?

Reply:
Hmm... I'd say so. Example:

```js
entry: {
  type: 'sendMessage',
  message: 'hello'
}
```
And...
```js
actions: {
  sendMessage: [
    'sendEmail',
    'notifySent'
  ],
  sendEmail: ({ action }) => {
    sendEmail(action.params.message);
  },
  notifySent: ({ action }) => {
    toast.send(`Message sent: ${action.params.message}`);
  }
}
```

Reply:
OH wow okay! Thanks for the example!

I didn't realize there was now an `action` key on action functions. That's beautiful and it makes sense why we'd want params here.

I added a (failing) test to make sure we're handling this and I'll work on getting that resolved.